specreduce/fluxcal.py

```diff
@@ -146,15 +146,15 @@ def onedstd(self, stdstar):
             directory to be used for the flux calibration. The user must provide the
             subdirectory and file name.
 
-            For example:
+            For example::
 
-            >>> standard_sensfunc(obj_wave, obj_flux, stdstar='spec50cal/bd284211.dat', mode='spline')  # doctest: +SKIP
```

Comment:
If there is no intention to run this through doctest, omitting `>>>` is enough to stop it from being collected by pytest-doctestplus.

Reply:
yeah, that's a pretty old example that, iirc, dates back to before `specreduce-data` was split into its own repository. it should definitely be skipped by doctest one way or the other. if omitting `>>>` is a preferred method, might as well do that here.